### ISSUE TITLE
[5.5.0] `xrdcks -h` seg faults

### ISSUE THREAD

**Issue by andresailer** [xrootd/xrootd#1780]
Reproducer:
```
source /cvmfs/sft-nightlies.cern.ch/lcg/views/dev4/latest/x86_64-centos7-gcc11-dbg/setup.sh
xrdcks -h
Segmentation fault (core dumped)
```

https://github.com/xrootd/xrootd/blob/b473acfa960811be3d076e687463dc7f04be3ecf/src/XrdApps/XrdCks.cc#L115-L118

I think line 115 should be
```c++
if(argc <= 2)
```

gdb (snipped some stuff)
```
$ gdb --args xrdcks -h
GNU gdb (GDB) 11.2
(gdb) run
Starting program: /cvmfs/sft-nightlies.cern.ch/lcg/views/dev4/Tue/x86_64-centos7-gcc11-dbg/bin/xrdcks -h

Program received signal SIGSEGV, Segmentation fault.
0x00007ffff766e8c1 in __strlen_sse2_pminub () from /lib64/libc.so.6
(gdb) bt
#0  0x00007ffff766e8c1 in __strlen_sse2_pminub () from /lib64/libc.so.6
#1  0x00000000004011b7 in XrdCksData::Set (csName=0x0, this=0x404420 <xCS>) at /build/jenkins/workspace/lcg_nightly_pipeline/build/externals/xrootd-5.5.0/src/xrootd/5.5.0/src/./XrdCks/XrdCksData.hh:82
#2  main (argc=2, argv=0x7fffffffa648) at /build/jenkins/workspace/lcg_nightly_pipeline/build/externals/xrootd-5.5.0/src/xrootd/5.5.0/src/XrdApps/XrdCks.cc:123
```

**Comment by simonmichal** [comment]
@andresailer : thanks for reporting this problem, from what I see @abh3 fixed the problem yesterday in: 16a28f7c4acf9b95dfb8c9477b5b95169a1100cf, would you like to give it a try?

**Comment by andresailer** [comment]
Hi @simonmichal 
Yes, that fixes the problem.
Thanks!
